src/charm.py

```diff
@@ -174,17 +118,17 @@ def _pebble_layer(self) -> Dict:
                     "user": "gh_exporter",
                     "command": "/srv/gh_exporter/github-actions-exporter",
                     "environment": {
```

Comment:
The usual way this is done is to add a function `get_env` or something like that in the module for the workload and delegate the names of the environment variables to that module, would that be appropriate here as well?

Reply:
Do you mean like the gosherve example in the spec?  I thought that this technique should be used for charms with more complexity. Do you think this is the case?

Reply:
Yes similar to that example, wouldn't that go into the exporter module that we already have?

Reply:
No problem, moved.